output/cloud/expv2/sink.go

```diff
@@ -1,13 +1,78 @@
 package expv2
 
 import (
+	"fmt"
+
 	"go.k6.io/k6/metrics"
 )
 
-func newSink(mt metrics.MetricType) metrics.Sink {
-	if mt == metrics.Trend {
-		return &histogram{}
+type aggregatedMetric interface {
```

Comment:
While I appreciate the intention to rename `Sink`, which got me confused more than once as I look at it, I find that `aggregatedMetric` is somewhat [tautological](https://dictionary.cambridge.org/dictionary/english/tautological) from my point of view. I think of metrics as aggregated views over specific classes of samples over time.

What would you think of `sampleAggregator`, `sampleAccumulator`, or `metricRunningValue` instead?

PS: absolutely not a blocker though 👍🏻

Reply:
I appreciate this suggestion, I picked `metricValue`. I removed _running_ because I think we don't really have a no-running metric value so it doesn't add any value, IMHO.

Let's reopen if you disagree.